### PR TITLE
Go through and cleanup Scenario 1

### PR DESCRIPTION
**Pull request by Keno**

### CODE REVIEW

File: docs/src/Scenario1/Evaluation_Scenario_1.md [hunk 1]
```diff
@@ -27,7 +56,6 @@ function make_statified_model(pops; pop_assumption = (stratum, pop) -> 1)
 
     for (i, nn) in enumerate(pops)
         defs[Ns[i]] = nn
-        # ASSUMPTION: Assume only one person in each age group is infectious at the beginning of the simulation
```
Comment (ChrisRackauckas) [inline docs/src/Scenario1/Evaluation_Scenario_1.md:30]: This is deleted but isn't mentioned in the text?

File: docs/src/Scenario1/Evaluation_Scenario_1.md [hunk 2]
```diff
@@ -46,53 +74,75 @@ function make_statified_model(pops; pop_assumption = (stratum, pop) -> 1)
...
+## Question 1
 
+> Start with a simple stratification with three age groups: young, middle-aged, and old.
+
+### Sub-question 1.a.
+> Begin with a situation where the population size across each age group is uniform: N_young = 2k, N_middle = 2k, N_old = 2k. Assume only one person in each age group is infectious at the beginning of the simulation. Let gamma = 1/14 days, and let R0 = 5. Assume gamma, beta, and R0 are the same for all age groups.
+
+> i. Simulate this model for the case where the 3x3 contact matrix is uniform (all values in matrix are 0.33)
+
+N.B.: Uniform `1/n_strata` is the default in our model creation function above.
```
Comment (ChrisRackauckas) [inline docs/src/Scenario1/Evaluation_Scenario_1.md:88]: N.B.?

File: docs/src/Scenario1/Evaluation_Scenario_1.md (hunk 1 above)
Comment (Keno) [reply docs/src/Scenario1/Evaluation_Scenario_1.md:30]: It's mentioned in the doc comment. The assumption was generalized per MITRE request to allow different assumptions in question 2.

File: docs/src/Scenario1/Evaluation_Scenario_1.md (hunk 2 above)
Comment (Keno) [reply docs/src/Scenario1/Evaluation_Scenario_1.md:88]: Nota bene

File: docs/src/Scenario1/Evaluation_Scenario_1.md [hunk 3]
```diff
@@ -46,53 +74,75 @@ function make_statified_model(pops; pop_assumption = (stratum, pop) -> 1)
...
 ```
 
-## Question 1
-
-> Simulate this model for the case where there is significant in-group contact
+> ii. Simulate this model for the case where there is significant in-group contact
 > preference – you may choose the numbers in the matrix to represent this in-
 > group preference.
 
+We change the contact matrix by having significant (0.3)
```
Comment (ChrisRackauckas) [inline docs/src/Scenario1/Evaluation_Scenario_1.md:101]: I don't get this: the resulting matrix doesn't have 0.3?

Comment (Keno) [reply docs/src/Scenario1/Evaluation_Scenario_1.md:101]: I changed it after writing that to make the plots more interesting. I'll update the comment.

File: docs/src/Scenario1/Evaluation_Scenario_1.md [hunk 4]
```diff
@@ -127,30 +177,52 @@ plot(plt1, plt2, plt3, plt4, plt5, size = (1000, 500))
...
+plot(plt_a5, plt_b5, plt_c5)
 ```
 
 ## Question 2
 
 > Now find real contact matrix data and stratify the basic SIR model with the appropriate number of age groups to match the data found. To simulate the model with realistic initial values, find data on population distribution by age group. As in question 1, let gamma = 1/14 days, and let R0 = 5. Assume gamma, beta, and R0 are the same for all age groups.
 
+TA1 provided the data from "Projecting social contact matrices in 152 countries using contact surveys and demographic data" by Prem, et al.
+This paper comes with 10 Excel files that provide contact matrices
+for 152 countries at work, home, school and other locations (plus
```
Comment (ChrisRackauckas) [inline docs/src/Scenario1/Evaluation_Scenario_1.md:216]: Link it?

File: docs/src/Scenario1/Evaluation_Scenario_1.md [hunk 5]
```diff
@@ -170,14 +242,31 @@ xfs2 = (; all = xf_all_locations2, work = xf_work2, school = xf_school2,
...
+Next we load the population distribution data.
 
-# Load Belgium population distribution
+```@example scenario1
 pop_belg = values(CSV.read("data/2022_ Belgium_population_by_age.csv", DataFrame,
                            header = 3)[1, 2:17])
+bar(1:length(pop_belg), collect(pop_belg), permute=(:x, :y), xlabel="Age (5 year buckets)", ylabel="Total # of people", leg=:none)
+```
 
+```
```
Comment (ChrisRackauckas) [inline docs/src/Scenario1/Evaluation_Scenario_1.md:269]: This part isn't ran

File: docs/src/Scenario1/Evaluation_Scenario_1.md (hunk 1 above)
Comment (ChrisRackauckas) [reply docs/src/Scenario1/Evaluation_Scenario_1.md:30]: We should probably mention that in the text then.

File: docs/src/Scenario1/Evaluation_Scenario_1.md (hunk 2 above)
Comment (ChrisRackauckas) [reply docs/src/Scenario1/Evaluation_Scenario_1.md:88]: Fancy.

File: docs/src/Scenario1/Evaluation_Scenario_1.md (hunk 4 above)
Comment (Keno) [reply docs/src/Scenario1/Evaluation_Scenario_1.md:216]: sure

**Comment by Keno** [comment]
Updated.

**Review by ChrisRackauckas — APPROVED** [review]

**Pull request merged**